clickstream/src/main/kotlin/clickstream/internal/eventscheduler/CSEventScheduler.kt

```diff
@@ -144,12 +161,29 @@ internal open class CSEventScheduler(
 
             val (eventData, eventHealthData) = CSEventData.create(event)
             eventHealthListener.onEventCreated(eventHealthData)
+            dispatchEventToInterceptor(
```

Comment:
would be preferable to not execute the function at all if `eventInterceptors` is empty